extension/src/popup/components/WarningMessages/index.tsx

```diff
@@ -365,3 +374,188 @@ export const ScamAssetWarning = ({
+        setErrorAsset(getCanonicalFromAsset(code, issuer));
+        navigateTo(ROUTES.trustlineError);
+      }
+    }
+  };
+
+  return (
+    <div className="NewAssetWarning">
+      <div className="NewAssetWarning__wrapper" ref={warningRef}>
+        <div className="NewAssetWarning__header">
+          {t("Before You Add This Asset")}
+        </div>
+        <div className="NewAssetWarning__description">
+          {t(
+            "Please double-check its information and characteristics. This can help you identify fraudulent assets.",
+          )}
+        </div>
+        <div className="NewAssetWarning__row">
+          <ManageAssetRow
+            code={code}
+            issuer={issuer}
+            image={image}
+            domain={domain}
+          />
+        </div>
+        <hr className="NewAssetWarning__list-divider" />
+        <div className="NewAssetWarning__flags">
+          {isRevocable && (
+            <div className="NewAssetWarning__flag">
+              <div className="NewAssetWarning__flag__icon">
+                <img src={IconShieldCross} alt="revocable" />
+              </div>
+              <div className="NewAssetWarning__flag__content">
+                <div className="NewAssetWarning__flag__header">
+                  {t("Revocable Asset")}
+                </div>
+                <div className="NewAssetWarning__flag__description">
+                  {t(
+                    "The asset creator can revoke your access to this asset at anytime",
+                  )}
+                </div>
+              </div>
+            </div>
+          )}
+          <div>
+            {isNewAsset && (
+              <div className="NewAssetWarning__flag">
+                <div className="NewAssetWarning__flag__icon">
+                  <img src={IconInvalid} alt="new asset" />
+                </div>
+                <div className="NewAssetWarning__flag__content">
+                  <div className="NewAssetWarning__flag__header">
+                    {t("New Asset")}
+                  </div>
+                  <div className="NewAssetWarning__flag__description">
+                    {t("This is a relatively new asset.")}
+                  </div>
+                </div>
+              </div>
+            )}
+          </div>
+          <div>
+            {isInvalidDomain && (
+              <div className="NewAssetWarning__flag">
+                <div className="NewAssetWarning__flag__icon">
+                  <img src={IconWarning} alt="invalid domain" />
+                </div>
+                <div className="NewAssetWarning__flag__content">
+                  <div className="NewAssetWarning__flag__header">
+                    {t("Invalid Format Asset")}
+                  </div>
+                  <div className="NewAssetWarning__flag__description">
+                    {t(
+                      "Asset home domain doesn't exist, TOML file format is invalid, or asset doesn't match currency description",
```

Comment:
```suggestion
                      "Asset home domain doesn‘t exist, TOML file format is invalid, or asset doesn't match currency description",
```

Reply:
thx for catching these i always forget lol 
https://github.com/stellar/freighter/pull/634/commits/a56954289da8e76c17a1d984d6d54ece5909c2a7